numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -1088,20 +1087,12 @@ def f(size):
             with assert_dpnp_implementaion():
                 with dpctl.device_context("opencl:gpu"):
                     result = f(size)
+                    # TODO: check result, x belongs R
 
-            if np.isscalar(result):
-                self.assertTrue(result >= 0.0)
-            else:
-                final_result = result.ravel()
-                self.assertTrue(final_result.all() >= 0.0)
-
-    @unittest.skip(
-        "TypeError: dpnp_random_impl() got an unexpected keyword argument 'low'"
-    )
     def test_uniform(self):
         @njit
         def f(low, high, size):
-            res = np.random.standard_normal(low=low, high=high, size=size)
+            res = np.random.uniform(low=low, high=high, size=size)
```

Comment:
Yes, test should pass.

In the provided example you compare boolean with float.
```
print(np.all(a) >= low)
print(np.all(a) < high)
```
We need to modify it in this way like I do in tests:

```
print(np.all(a >= low))
print(np.all(a < high))
```

Functions [np.all](https://numpy.org/doc/stable/reference/generated/numpy.all.html) and [np.ndarray.all](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.all.html#numpy.ndarray.all) are equal. Only the difference is that `np.all()` makes cast to np.array to call it for every array-like type. Calling np.ndarray.all may be slightly faster because it is built-in function for numpy arrays. I can modify tests if needed.

```
print((a >= low).all())
print((a < high).all())
```

Reply:
That makes perfect sense. Thank you for your input. In that case I think we do not need to make any changes to the checks.